src/containers/AccountModal/forms/EditPassword.tsx

```diff
@@ -40,8 +41,8 @@ const ResetPassword: React.FC = () => {
         }
         await changePasswordWithToken(emailParam || '', password, resetToken, passwordConfirmation);
       }
-      await logout();
-      navigate(addQueryParams(window.location.origin, { u: 'login' }));
+      await logout(true);
```

Comment:
Why do we not actually log the user out here? Have you tested this change with the Cleeng integration?